README.md

```diff
@@ -8,5 +8,10 @@
 >
 >> install streamlit using command below:
 >> - `pip install streamlit`
```

Comment:
🧗‍♀️ if you are using requirements.txt file it is better to change this section using the command below:
`pip install -r requirements.txt`